modules/library/src/main/scala/zio/elasticsearch/aggregation/Aggregations.scala

```diff
@@ -32,6 +33,61 @@ sealed trait ElasticAggregation { self =>
 
 sealed trait SingleElasticAggregation extends ElasticAggregation
 
+sealed trait BucketSortAggregation extends SingleElasticAggregation with HasSize[BucketSortAggregation] with WithAgg {
+
+  /**
+   * Sets the starting offset from where the [[zio.elasticsearch.aggregation.BucketSortAggregation]] return results.
```

Comment:
Strange sentence, can we rephrase it?

Reply:
Aggred to replace `return` with `returns`.